File: JAVA SCRIPT/19.js
//19. Check if the array arr = [1, 2, 3, 4, 5] is sorted in ascending order, and print true or false.
var arr=[1,2,4,3,5];
var sorted=[...arr].sort((a, b) => a - b); 
var count=0;
for(i=0;i<arr.length;i++){
    if(arr[i]==sorted[i]){
        count++;
    }
}
if(count==arr.length){
    console.log("true")
}
else{
    console.log("false");
}

File: C LANGUAGE/19.c
//19. Check if the array arr = [1, 2, 3, 4, 5] is sorted in ascending order, and print true or false.
#include<stdio.h>
int main(){
	int arr[]={1,2,4,3,5};
	int i,j,k;
	int length=sizeof(arr)/sizeof(arr[0]);
	for(i=0;i<length;i++){
		for(j=i+1;j<length;j++){
			if(arr[i]>arr[j]){
				int sum=arr[i]+arr[j];
				arr[i]=arr[j];
				arr[j]=sum-arr[i];
			}	
		}
	}
	for(k=0;k<length;k++){
	printf("%d",arr[k]);	
	}
	
}
